src/backend/core/tests/teams/test_core_api_teams_retrieve.py

```diff
@@ -1,16 +1,12 @@
 """
 Tests for Teams API endpoint in People's core app: retrieve
 """
-import random
-from collections import Counter
-from unittest import mock
-
```

Comment:
do we have a linter configuration problem? This should have been raised by pylint right?

Reply:
but the CI linter should have failed... the lint-back job in the CI doesn't seem to cover tests :thinking:

Reply:
From what I see on that other PR, it seems to be working as expected : https://github.com/numerique-gouv/people/actions/runs/7818569220/job/21329100153?pr=44